resources/guides/Navigation Guide/navigation-guide.mdx

```diff
@@ -0,0 +1,80 @@
+
+![The Wiki's Navigation Bar](/img/nav_guide/nav_bar.jpg)
+
+### The Section's Navigation Bar
+
+When you click a link in the main navigation bar, you'll see a navigation bar on the left of the website along with the index page of the section where you'll find the links to all the subpages that relate to the selected section. 
+
+![The Index Page](/img/nav_guide/index_page.jpg)
+
+By clicking on a link in the section's navigation bar or a link on the index page, the Wiki will direct you to a page with more details regarding said link.
+
+### The Page's Navigation Bar
+
+Once an article/page has been loaded, you can see the page's navigation bar on the right side of the website. You can use this to navigate the page.
+
+![Page Navigation Bar](/img/nav_guide/ex_of_page_navbar.jpg)
+
+### The Search Bar
+
+You can also use the search bar on the top right corner of the website. The search feature of the Wiki will display pages that contain the word or question you intend on searching. When you click on one of the results, the Wiki will direct you to the page.
+
+![The Search Bar](/img/nav_guide/search_bar.jpg)
+
+### Footer
+
+Here, you'll find links to our social media, our discord server and other external links.
+
+## Sections of the Wiki
+
+### Courses
+
+Here, you'll find everything you need to know regarding courses. We provide an interactive table where you can view **what term the courses are offered in along with prerequisites and what's mandatory for your program**. We also provide an **overview of the courses along with advice from previous students** in order to better prepare you.
+
+### Careers
+
+Here, you'll find information regarding **CO-OP and internships** as well as a guide on **creating resumes, cover letters and interviews**.
+
+### Academics
+
+Here, you'll find resources regarding the **services and opportunities the University offers** such as help centers, financial aid and awards.
+
+### Resources
+
+Here, you'll find **how-to guides regarding using UWindsor services** along with **advice and tips** for a successful university career.
+
+### CSS
+
+Here, you'll find information about the **Computer Science Society** and the **Computer Science Society Student Council**.
+
+### Newsletter
+
+Checkout some articles and stories that Computer Science students have written!
+
+## FAQ
+
++ [How to enroll in classes?](https://uwindsorcss.github.io/wiki/resources/guides/uwinsite_guide#how-to-plan--enroll-in-classes)
++ [What is the difference between CO-OP and internship?](https://uwindsorcss.github.io/wiki/careers/choosing)
++ [I'm a first year and need guidance on how to get started.](https://uwindsorcss.github.io/wiki/resources/guides/first_year)
++ [What courses should I take? What electives are available?](https://uwindsorcss.github.io/wiki/courses/course_planning/recommendedsequence#recommended-sequence)
+
+
+## Quick Links
+
+import App from './nav-guide-generator'
```

Comment:
I think putting `WikiPages` component instead of `App` component is the best idea.